### PR TITLE
Updates to vis documentation

### PR DESCRIPTION
**Pull request by srizzi88**
* Bringing back Paraview tutorial
* Bringing back manual launching for Paraview
* General fixes

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -90,6 +90,8 @@ nav:
         - Visit: polaris/visualization/visit.md
         - FFmpeg: polaris/visualization/ffmpeg.md
         - ImageMagick: polaris/visualization/imagemagick.md
+        - Paraview tutorial: polaris/visualization/paraview-tutorial.md
```
Comment (felker) [inline mkdocs.yml:93]: Looks like it should be capitalized as "ParaView" everywhere?

Comment (josephinsley) [reply mkdocs.yml:93]: Yes.  And we should ensure that "VisIt" is consistent, as well.

Comment (srizzi88) [reply mkdocs.yml]: Updated now. Thanks for the feedback

**Review by josephinsley — APPROVED** [review]
updates and new content looks good

**Pull request merged**